odl/operator/oputils.py

```diff
@@ -133,8 +134,8 @@ def power_method_opnorm(op, xstart=None, maxiter=100, rtol=1e-05, atol=1e-08,
         `Operator.adjoint` must be defined (which implies that the
         operator must be linear).
     xstart : ``op.domain`` `element-like`, optional
-        Starting point of the iteration. By default, the ``one``
-        element of the `Operator.domain` is used.
+        Starting point of the iteration. By default an `Operator.domain`
+        element containing noise is used.
```

Comment:
Maybe this change makes the Python 2.7 doctest fail, and a non-random starting point should be used in the test?

Reply:
Probably. We need to change the doctest a bit then.